docs/api/v1.19.md

```diff
@@ -16,9 +16,7 @@ redirect_from:
      will be rejected.
 -->
 
-# Docker Engine API v1.19
-
-## 1. Brief introduction
+# 1. Brief introduction
```

Comment:
This needs to stay a H2. The H1 is generated from the metadata at the top (Looks like the other versions have to be updated accordingly)